component/componenttest/nop_telemetry.go

```diff
@@ -26,5 +27,6 @@ func NewNopTelemetrySettings() component.TelemetrySettings {
 	return component.TelemetrySettings{
 		Logger:         zap.NewNop(),
 		TracerProvider: trace.NewNoopTracerProvider(),
+		MeterProvider:  metric.NoopMeterProvider{},
```

Comment:
Please file an issue in otel-go that this is inconsistent with TracerProvider.

Reply:
This was already addressed here https://github.com/open-telemetry/opentelemetry-go/commit/fe7058daa265d0e7d04ecd045f796ff04c54da8f